<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Floor</class>
 <widget class="QWidget" name="Floor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>98</width>
    <height>91</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>98</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>98</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="floorLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="upButton">
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="downButton">
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
